content/docs/04.5-devx/04-resource-quota.md

```diff
@@ -34,12 +34,11 @@ The developer quota can be managed through the Palette Platform as below:
 
 3. Set the `User Quota,` where default values are as follows:
```

Comment:
```suggestion
3. Set the desired *User Quotas* values.  The user quotas have the following default values:
```
Because we are not referring to the UI element but rather the label, we can use italics the first time.